.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+**Organization**:
```

Comment:
@jimfenton why is this being ported?  I thought it has to live in master?  Or does it just have to live in default branch?